db/migrate/20240412024910_create_currencies.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCurrencies < ActiveRecord::Migration[7.0]
+  def change
+    create_table :currencies, id: :uuid do |t|
+      t.string :name
+      t.string :symbol
+      t.uuid :company_id
```

Comment:
For all relationships, migration does not create the foreign key at the database level
We might need to create them explicity by using `t.reference`, `t.references` or `add_foreign_key` method